src/main/external-resources/UMS.conf

```diff
@@ -196,7 +196,7 @@ http_engine_v2 =
 # Prevent system idle sleep
 # -------------------------
 # If your UMS host has a power-saving mode, use this option to prevent the
-# system from hibernating or sleeping. Currently only Windows and macOS are
+# system from hibernating or sleeping. Currently only Windows and Mac OS are
 # supported. There are three modes:
```

Comment:
@Sami32 The current OS was never called ```Mac OS```. The OS made by Apple was called ```Mac OS```, but they abandoned it with version 9 - and that OS has little in common with the current ```macOS```. The current OS, which is a "heavily modified version of BSD", was first called ```Mac OS X```. In 2012 it was renamed to ```OS X``` - which was it's name until they renamed it again in 2016 to ```macOS```. 

It's always a sure sign that there's no real innovation when companies constantly keep renaming their product.